src/compas_fab/robots/robot.py

```diff
@@ -1125,25 +1159,27 @@ def send_trajectory(self):
         self.ensure_client()
         raise NotImplementedError
 
-    @property
-    def frames(self):
-        """The robot's frames."""
-        return self.model.frames
+    def updated_frames(self, configuration, group=None):
```

Comment:
Is it really `updated_frames` the correct name for this? Shouldn't it be `transformed_frames`? Or `current_frames` even? Not entirely sure...

Reply:
selected `transformed_frames`